docs/porting/libraries.md

```diff
@@ -1,5 +1,311 @@
+
+### Mixing approaches
+
+It's likely that you'll mix the above approaches on a per-project basis.  You should do what makes the most sense to you and for your codebase.
+
+## Picking your project system in Visual Studio: xproj or traditional projects
+
+In addition to a new project model, .NET Core introduces a new project system, xproj, in Visual Studio.  You'll have to pick the correct project system based on your needs.
+
+### When to pick xproj
+
+The new Xproj project system utilizes the capabilities of the `project.json`-based project model to offer two major features over existing project types: seamless multitargeting by building multiple assemblies and the ability to directly generate a NuGet package on build.
+
+However, it comes at the cost of lacking certain features you may use, such as:
+
+- Support for F# or VB
+- Generating satellite assemblies with localized resource strings
+- Referencing a `.dll` file on the filesystem
+
+If your project needs are relatively minimal and you can take advantage of the new features of xproj, you should pick it as your project system.  This can be done in Visual Studio as such:
+
+1. File | New Project.
+2. Select ".NET Core" under Visual C#.
+3. Selecting the "Class Library (.NET Core)" template. 
+
+### When to pick a traditional project system
+
+You can target .NET Core with the traditional project system in Visual Studio, by creating a Portable Class Library (PCL) and selecting ".NET Core" in the project configuration dialog.
+
+If you have more intricate project system needs, this should be your choice.  It may also be the best choice if you're mixing .NET Core and .NET Framework code in a larger solution.  Note that if you wish to multitarget by generating platform-specific assemblies like with the `xproj` project system, you'll need to create a "Bait and Switch" PCL, [as described here](https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/).
+
+## Porting your tests (if applicable)
+
+The best way to make sure everything works when you've ported your code is to *test your code as you port it to .NET Core*.  To do this, you'll need to use a testing framework that will build and run tests for .NET Core.  Currently, you have two options:
+
+* [xUnit](https://xunit.github.io/)
+   - [Getting Started](http://xunit.github.io/docs/getting-started-dnx.html)
+   - [Tool to convert an MSTest project to xUnit](https://github.com/dotnet/codeformatter/tree/master/src/XUnitConverter)
+* [NUnit](http://www.nunit.org/)
+  - [Getting Started](https://github.com/nunit/docs/wiki/Installation)
+  - [Blog post about migrating from MSTest to NUnit](http://www.florian-rappl.de/News/Page/275/convert-mstest-to-nunit)
+  
+Currently, MSTest does not run on .NET Core.  If you're using MSTest, you'll have to migrate to xUnit or NUnit.  If you've been using Xunit or Nunit to write your tests for your library, great!
```

Comment:
You could ask Daniel directly. I have it marked as done, but maybe that's my mistake.